examples/nlp/question_answering/conf/config.yaml

```diff
@@ -65,15 +64,75 @@ model:
     # generated. This is needed because the start
     # and end predictions are not conditioned
     # on one another.
-    output_prediction_file: predictions.json
+    output_prediction_file: null
+    # File to write predictions to. Only in evaluation or test mode,
+    output_nbest_file: null
+    # File to write nbest predictions to. Only in evaluation or test mode.
+    null_score_diff_threshold: 0.0
+    # If null_score - best_non_null is greater than the threshold predict null.
+    n_best_size: 20
+    # The total number of n-best predictions to generate at testing.
+
+  test_ds:
+    mode: test
+    file: null # .json file
+    batch_size: 24 # per GPU
+    shuffle: false
+    num_samples: -1
+    num_workers:  2
+    pin_memory: false
+    drop_last: false
+    max_answer_length: 30
+    # The maximum length of an answer that can be
+    # generated. This is needed because the start
+    # and end predictions are not conditioned
+    # on one another.
+    output_prediction_file: test_prediction.json
     # File to write predictions to. Only in evaluation or test mode,
-    output_nbest_file: nbest.json
+    output_nbest_file: test_nbest.json
     # File to write nbest predictions to. Only in evaluation or test mode.
     null_score_diff_threshold: 0.0
     # If null_score - best_non_null is greater than the threshold predict null.
     n_best_size: 20
     # The total number of n-best predictions to generate at testing.
 
+    
+  tokenizer:
+    tokenizer_name: nemobert # or sentencepiece
+    pretrained_model_name: ${model.pretrained_model_name}
+    special_tokens: null
+    vocab_file: null # path to vocab file
+    data_file: null # used to build sentencepiece
+    sample_size: null # for sentencepiece
+    vocab_size: null # vocab size for building sentence piece
+    do_lower_case: ${model.dataset.do_lower_case} # only used when tokenizer is build with vocab file or with sentencepiece  
+    tokenizer_model: null # only used if tokenizer is sentencepiece
+
+  language_model:
+    pretrained_model_name: ${model.pretrained_model_name}
+    bert_checkpoint: null
+    bert_config: null # loads pretrained model. To initialize model from scratch uncomment the following
+      # attention_probs_dropout_prob: 0.1
+      # hidden_act: gelu
+      # hidden_dropout_prob: 0.1
+      # hidden_size: 768
+      # initializer_range: 0.02
+      # intermediate_size: 3072
+      # max_position_embeddings: 512
+      # num_attention_heads: 12
+      # num_hidden_layers: 12
+      # type_vocab_size: 2
+      # vocab_size: 30522
+    bert_config_file: null # json file, precedence over bert_config
```

Comment:
do you need these comments there?

Reply:
I would like to add these, so the user knows how to instantiate a BERT model from scratch with different layout